app/src/main/java/org/p2p/wallet/home/repository/RefreshErrorRepository.kt

```diff
@@ -0,0 +1,8 @@
+package org.p2p.wallet.home.repository
+
+import kotlinx.coroutines.flow.Flow
+
+interface RefreshErrorRepository {
+    fun getFlow(): Flow<Unit>
```

Comment:
getFlow of what?

Reply:
renamed to:
getRefreshClickedFlow()
notifyRefreshClicked()